dotnet/src/Connectors/Connectors.GoogleVertexAI/Extensions/VertexAIKernelBuilderExtensions.cs

```diff
@@ -170,7 +176,10 @@ public static class VertexAIKernelBuilderExtensions
     /// </summary>
     /// <param name="builder">The kernel builder.</param>
     /// <param name="modelId">The model for text generation.</param>
-    /// <param name="bearerKeyProvider">The Bearer Key provider for authentication.</param>
+    /// <param name="bearerKeyProvider">
```

Comment:
Update the comment to same as above but instead of within `param`. 

Add this to a `<remarks>` tag after the `<summary>`.

Content for `<remarks>`: 

```csharp
/// <remarks>
/// This <paramref name="bearerKeyProvider"/> will be called on every request, 
/// when providing the token consider using caching strategy and refresh token logic 
/// when it is expired or close to expiration.
/// </remarks>
```

Reply:
Good, it's cleaner.